### ISSUE TITLE
Hotkey doesn't register & hotkey bar display issue

### ISSUE THREAD

**Issue by lonely701** [Belphemur/SoundSwitch#429]
Hotkey win+shift+ins cannot register at version 5.0.3. I noticed #421, but it's not completely solved. Also, the hotkey bar is too short since 5.0.1. I am using a 4k screen maybe that's related.
**Neither of the two issues occurred to v5.0.1.** (I noticed since then it shows LWin or RWin instead of Win)

![image](https://user-images.githubusercontent.com/5812690/80935789-40cfdb80-8d9c-11ea-9352-2df919839755.png)
![image](https://user-images.githubusercontent.com/5812690/80935880-a4f29f80-8d9c-11ea-88ec-d2acc1e1457e.png)

**Comment by Belphemur** [comment]
Can you try to uninstall and reinstall the latest version ?

I have the feeling the updater didn't do its job correctly. The field shouldn't look like that.

**Comment by lonely701** [comment]
I reinstalled it with "remove existing settings" checked. Problem remains. It looks ok on my 2k external monitor, so I guess it's not optimized for 4k. (I signed out and sign back in when I switch between different resolutions.)

And the hotkey issue is still there. Win+ins does register, but win+shift+ins doesn't work. When I press and hold win and shift key, it shows green texts in the input field, then I press ins and release all of the three, everything is gone in the field. There are no hotkey conflicts or notifications. I thought it was a display issue. I closed the setting and tried it. Still didn't work.
